app/models/person.rb

```diff
@@ -12,6 +12,7 @@
 #  active_days_count                  :integer          default(0)
 #  last_page_load_date                :datetime
 #  test_group_number                  :integer          default(1)
+#  active                             :boolean          default(TRUE)
```

Comment:
I guess you have new fields in database that these migrations don't touch? Seems a bit odd that there's new fields added to annotations even though they are not present in the migrations themselves.